Code/Framework/AzCore/Platform/Windows/AzCore/PlatformIncl_Windows.h

```diff
@@ -9,6 +9,7 @@
 #pragma once
 
 #define WIN32_LEAN_AND_MEAN
+#define UNICODE
```

Comment:
Marking this with a comment to indicate that it is temporary

Reply:
is not temporary, we should always call the UNICODE versions according to @lawsonamzn 
This makes it that e.g. `CreateProcess` calls `CreateProcessW` instead of `CreateProcessA`

Reply:
We shouldn't be defining UNICODE at all.
We should just call the W versions and not the macro versions.

Reply:
That is really hard to find/fix and will continue being error prone. Someone calling `CreateProcess` will call the non-unicode version without being fully aware. If this is kept, they will have to call `CreateProcessA` instead, which is more obvious/easy to spot and fix.
UNICODE also changes the value that e.g. TCHAR is and will change all the "T" based macros/types. 
I dont disagree that the ideal approach would be to everything use UNICODE, I just dont think that is practical. Defining UNICODE catches some of the cases (not all, you can still call the A versions).